numpy/_core/fromnumeric.py

```diff
@@ -2210,12 +2210,17 @@ def compress(condition, a, axis=None, out=None):
     return _wrapfunc(a, 'compress', condition, axis=axis, out=out)
 
 
-def _clip_dispatcher(a, a_min, a_max, out=None, **kwargs):
-    return (a, a_min, a_max)
+_NotPassed = object()
+
+
+def _clip_dispatcher(a, a_min=None, a_max=None, out=None, *, min=None,
+                     max=None, **kwargs):
+    return (a,)
 
 
 @array_function_dispatch(_clip_dispatcher)
-def clip(a, a_min, a_max, out=None, **kwargs):
+def clip(a, a_min=_NotPassed, a_max=_NotPassed, out=None, *, min=_NotPassed,
```

Comment:
Maybe just use `np._NoValue` - that is meant for cases like these (and looks better in the documentation!)

Reply:
Done!